request-management-api/request_api/services/documentservice.py

```diff
@@ -145,6 +138,10 @@ def uploadpersonaldocuments(self, requestid, attachments):
                 attachmentlist.append(attachmentresponse)
                 
             documentschema = CreateDocumentSchema().load({'documents': attachmentlist})
+            for document in documentschema['documents']:
+                # Add attachment event here as we need to pass in the document
+                # to the event service to identify if the document is an RRT document.
+                attachmenteventresponse = attachmentevent().createattachmentevent(requestid, userid, document)
             return self.createrequestdocument(requestid, documentschema, None, "rawrequest")        
```

Comment:
As discussed, attachmentevent call should be at resource level not at the service level.